examples/dti_prediction/main.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+
+import pytorch_lightning as pl
+from config import get_cfg_defaults
+from dataset import DTIDeepDataset
+from model import get_model
+from torch.utils.data import DataLoader
+
+
+def arg_parse():
+    """Parsing arguments"""
+    parser = argparse.ArgumentParser(description='DeepDTA on BindingDB dataset')
```

Comment:
'DeepDTA on BindingDB dataset': the example name should be a short version of this rather than dti_prediction